graylog2-server/src/main/java/org/graylog2/rest/resources/alerts/AlertResource.java

```diff
@@ -78,7 +78,7 @@ public AlertListSummary listAll(@ApiParam(name = "since", value = "Optional para
         List<AlertSummary> alerts = streamService.loadAll().stream()
                 .filter(stream -> isPermitted(RestPermissions.STREAMS_READ, stream.getId()))
                 .flatMap(stream -> alertService.loadRecentOfStream(stream.getId(), since).stream())
-                .limit(AlertImpl.MAX_LIST_COUNT)
+                .limit(AlertService.MAX_LIST_COUNT)
```

Comment:
This should be configurable, e. g. use a query parameter `int limit` with [`@DefaultValue("300")`](https://docs.oracle.com/javaee/7/api/javax/ws/rs/DefaultValue.html).